### PR TITLE
[CDS-100759] CD release notes 1.55.x

### PR DESCRIPTION
**Pull request by vishal-av**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by krishi0408 — DISMISSED** [review]

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -47,6 +47,31 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+- **Multiple runtime support for Serverless.com and AWS SAM**
+
+Released new images for AWS SAM to support multiple runtime for `python3.11`, `python3.12`, `ruby3.2`, `java8`, `go1.24`. For more information, go to Harness [AWS SAM - Containerized step images](/docs/continuous-delivery/deploy-srv-diff-platforms/aws/aws-sam-deployments/#containerized-step-images). (CDS-80560)
+
+Released new images for Serverless to support multiple runtime for `python3.11`, `python3.12`, `ruby3.2`, `java8`.  For more information, go Harness [Serverless.com - Containerized step images](/docs/continuous-delivery/deploy-srv-diff-platforms/serverless/serverless-lambda-cd-quickstart#containerized-step-images). (CDS-80560)
+
+- **Multi-Deployment and Matrix Failure Strategy Support**
+
+Harness now supports applying failure strategies during multi-service, multi-infrastructure, and matrix deployments. Currently this feature is behing the Feature Flag `CDS_MULTI_DEPLOYMENT_ON_FAILURE`. Please contact [Harness support](mailto:support@harness.io) to enable this feature. (CDS-96876, ZD-63199, ZD-64391, ZD-64465, ZD-66720)
```
Comment (SushrutHarness) [inline release-notes/continuous-delivery.md:62]:
```suggestion
Harness now supports applying failure strategies during multi-service, multi-infrastructure, and matrix deployments. Currently this feature is behind the feature flag `CDS_MULTI_DEPLOYMENT_ON_FAILURE`. Please contact [Harness support](mailto:support@harness.io) to enable this feature. (CDS-96876, ZD-63199, ZD-64391, ZD-64465, ZD-66720)
```

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -47,6 +47,31 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+- ServiceNow access is enabled through Mulesoft endpoint by adding `/now` URI in the ServiceNow requests. (CDS-99537, ZD-64547)
```
Comment (SushrutHarness) [inline release-notes/continuous-delivery.md:64]:
```suggestion
- ServiceNow access is enabled through a Mulesoft endpoint by adding a `/now` URI in the ServiceNow requests. (CDS-99537, ZD-64547)
```

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -47,6 +47,31 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+#### Fixed issues
+
+- Users were unable to move or delete a service-specific override. This issue is resolved. The permissions required to move or delete a service-specific override for Global Environment and infrastructure is `core_environment_edit`. The permissions required for service-specific override or service and infrastructure is `core_service_edit`. Separate permissions to delete is not required.  (CDS-100204, ZD-68468, ZD-68759)
```
Comment (SushrutHarness) [inline release-notes/continuous-delivery.md:68]:
```suggestion
- Users were unable to move or delete a service-specific override. This issue is resolved. The permissions required to move or delete a service-specific override for global environment and infrastructure is `core_environment_edit`. The permissions required for service-specific override or service and infrastructure is `core_service_edit`. Separate permissions to delete are not required.  (CDS-100204, ZD-68468, ZD-68759)
```

File: release-notes/continuous-delivery.md [hunk 4]
```diff
@@ -47,6 +47,31 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+- The service variable that referenced secret text was exposed in plaintext delegate logs when executing CD WinRM deployment. This issue is resolved. The warning logs are updated with debug logs. (CDS-100046, ZD-68713)
+- When using WinRM credential of "type = Kerberos" in PowerShell script, the output variables were not displayed properly due to a missing identifier. This issue is resolved. (CDS-100036, ZD-68283)
```
Comment (SushrutHarness) [inline release-notes/continuous-delivery.md:70]:
```suggestion
- When using a WinRM credential of "type = Kerberos" in a PowerShell script, the output variables were not displayed properly due to a missing identifier. This issue is resolved. (CDS-100036, ZD-68283)
```

File: release-notes/continuous-delivery.md [hunk 5]
```diff
@@ -47,6 +47,31 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+- Service was not loading when the service and environment used a custom template configured as **Always use the stable version**. This issue is resolved. (CDS-100021, ZD-68666)
+- Custom stage pipelines did not show the environment in the Pipeline execution history. This issue is resolved now.(CDS-99938)
```
Comment (SushrutHarness) [inline release-notes/continuous-delivery.md:72]:
```suggestion
- Custom stage pipelines did not show the environment in the pipeline execution history. This issue is resolved now.(CDS-99938)
```

File: release-notes/continuous-delivery.md [hunk 6]
```diff
@@ -47,6 +47,31 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+- The tooltip for the Output alias (optional) field under Optional Configuration was not being displayed. This issue is resolved. (CDS-90919)
```
Comment (SushrutHarness) [inline release-notes/continuous-delivery.md:73]:
```suggestion
- The tooltip for the **Output Alias** (optional) field under **Optional Configuration** was not being displayed. This issue is resolved. (CDS-90919)
```

Comment (SushrutHarness) [reply release-notes/continuous-delivery.md:73]: Please take a look at this again, if something is a UI element or displayed text on the UI and we're referring to that text, it should be bold.

**Review by SushrutHarness — COMMENTED** [review]
LGTM, minor nits

**Review by krishi0408 — DISMISSED** [review]

**Pull request merged**